<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Listmusicitemwi</class>
 <widget class="QWidget" name="Listmusicitemwi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMusicItem_wi" native="true">
     <property name="minimumSize">
      <size>
       <width>563</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>563</width>
       <height>90</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widgetMusicItem_wi {
	border-radius: 15px;
}
#widgetMusicItem_wi:hover{
	background-color: #ffffff;
 border: 1px solid #cccccc;       /* 添加 1px 灰色实线边框 */
}</string>
     </property>
     <widget class="QPushButton" name="index_btn">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>25</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
         font-weight: 500;
        background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="icon_lb">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
   border-radius: 20px;
    background-color: transparent;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>173</x>
        <y>15</y>
        <width>270</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="songname_lb">
         <property name="styleSheet">
          <string notr="true">color: #283248;
        font-size: 21px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
         </property>
         <property name="text">
          <string>歌曲名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="composer_lb">
         <property name="styleSheet">
          <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
         </property>
         <property name="text">
          <string>作者</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="duration_lb">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>25</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
     font-weight: 500;
        background-color: transparent;</string>
      </property>
      <property name="text">
       <string>时长</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
